usr/local/bin/bastille

```diff
@@ -62,7 +63,7 @@ bastille_perms_check() {
 bastille_perms_check
 
 ## version
-BASTILLE_VERSION="0.10.20231013"
+BASTILLE_VERSION=b7d741b5cd3b0c758f0983fd9546e88fba0354d7
```

Comment:
This doesn't need to be included in the patch.

Reply:
OK, will keep it in mind for next time!

Reply:
@waseigo please then remove the change from your PR